### DISCUSSION TITLE
Show activity status of other users in game

### DISCUSSION THREAD

**Discussion by Sharikobe** [ppy/osu#21897] [Ideas]
When is the osu!bancho screen going to be implemented in lazer? I press f9 but it only shows me who is online (image attached). I can't see what map they are currently playing or what activity they are doing. Is this going to implemented soon?

![osu_2022-12-29_00-06-15](https://user-images.githubusercontent.com/121575748/209857604-7507e400-eaae-4b43-afce-79880f3caec2.jpg)

**Comment by peppy** [comment]
Yep, this is the goal! We just need server support for this.
